akd_mysql/src/mysql.rs

```diff
@@ -952,82 +960,92 @@ impl Storage for AsyncMySqlDatabase {
                 debug!("BEGIN MySQL get batch");
                 let mut conn = self.get_connection().await?;
 
-                let results =
-                    if let Some(create_table_cmd) = DbRecord::get_batch_create_temp_table::<St>() {
-                        // Create the temp table of ids
-                        let out = conn.query_drop(create_table_cmd).await;
-                        self.check_for_infra_error(out)?;
-
-                        // Fill temp table with the requested ids
-                        let mut tx = conn.start_transaction(TxOpts::default()).await?;
-                        tx.query_drop("SET autocommit=0").await?;
-                        tx.query_drop("SET unique_checks=0").await?;
-                        tx.query_drop("SET foreign_key_checks=0").await?;
-
-                        let mut fallout: Option<Vec<_>> = None;
-                        let mut params = vec![];
-                        for batch in key_set_vec.chunks(self.tunable_insert_depth) {
-                            if batch.len() < self.tunable_insert_depth {
-                                fallout = Some(batch.to_vec());
-                            } else {
-                                params.push(
-                                    DbRecord::get_multi_row_specific_params::<St>(batch).unwrap(),
-                                );
-                            }
-                        }
-
-                        // insert the batches of size = MYSQL_EXTENDED_INSERT_DEPTH
-                        if !params.is_empty() {
-                            let fill_statement = DbRecord::get_batch_fill_temp_table::<St>(Some(
-                                self.tunable_insert_depth,
+                let results = if let Some(create_table_cmd) =
```

Comment:
Is it possible to rewrite this with an early else for better readability? i.e., 

```
if else_condition {
  return None;
}
// Proceed with valid results value
```

Reply:
Not really a great way, we'd lose timing information with an early exit (debug! prompts).